web/views.py

```
@@ -6425,6 +6453,46 @@
     )
 
 
+@login_required
+def self_report_attendance(request, session_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `self_report_attendance`

(ANN201)